SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3040,6 +3048,20 @@ void USpatialNetDriver::RefreshActorDormancy(AActor* Actor, bool bMakeDormant)
 		UE_LOG(LogSpatialOSNetDriver, Verbose, TEXT("Unable to flush dormancy on actor (%s) without entity id"), *Actor->GetName());
 		return;
 	}
+	if (!Connection->GetCoordinator().HasEntity(EntityId))
+	{
+		USpatialActorChannel* ActorChannel = GetActorChannelByEntityId(EntityId);
+		if (ActorChannel == nullptr)
+		{
+			return;
+		}
+
+		if (ActorChannel->bCreatedEntity)
+		{
+			ActorSystem->AddEntityToRefreshDormancy(EntityId, bMakeDormant);
+			return;
+		}
```

Comment:
need to add an else here, and error as it's not valid to continue if we don't have the entity checked out